omod/src/main/webapp/resources/htmlforms/015-HMIS_ACP_ExposedInfantClinicalChart-EncounterPage.xml

```diff
@@ -0,0 +1,422 @@
+
+            <span id="birthDate"><lookup expression="patient.birthdate"/></span>
+        </span>
+    </div>
+  
+    <div class="row">
+        <div class="col-md-12">
+            <section>
+                <div class="card">
+                    <div class="card-header">Appointment</div>
+                    <div class="card-body">
+                        <div class="row">
+                            <div class="col-md-3">
+                                <label>EI Number:</label>
+                                <patient field="identifier" identifierTypeId="2c5b695d-4bf3-452f-8a7c-fe3ee3432ffe" required="true"/>
+                            </div>
+                            <div class="col-md-3">
+                                <label>Type of visit</label>
+                                <obs conceptId="162992"
+                                    answerConceptIds="162993,162994,162995,162996,162997,162998,162999,163000,163001,163002,163018,163019,163020,163021,163022"
+                                    answerLabels="Visit 1,Visit 2,Visit 3,Visit 4,Visit 5,Visit 6,Visit 7,Visit 8,Visit 9,Visit 10,Visit 11,Visit 12,Visit 13,Visit 14,Visit 15"
+                                    required="true"/>
+                            </div>
+                            <div class="col-md-3">
+                                <label>Date of Appointment</label>
+                                <obs conceptId="99443" allowFutureDates="true"/>
+                            </div>
+                            <div class="col-md-3">
+                                <label>Date of Visit</label>
+                                <encounterDate default="today" disallowMultipleEncountersOnDate="block"/>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-md-3">
+                                <label>Next Appointment Date</label>
+                                <obs id="5096" conceptId="5096" allowFutureDates="true"/>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                              
+                <div class="card">
+                    <div class="card-header">HIV test, Infant Feeding, Immunisation Codes and Growth Measures</div>
+                    <div class="card-body">
+                        <h4 class="card-title">HIV test</h4>
+                        <div class="row">
+                            <div class="col-md-4">
+                                <obs conceptId="99080 " answerConceptIds="163006,163007" style="radio" class="horizontal"/>
+                            </div>
+                        </div>
+                        
+                        <h4 class="card-title mt-4">Infant feeding</h4>
+                        <div class="row">
+                            <div class="col-md-4">
```

Comment:
@semujju When you add an md4 I think you should add an empty md8 to ensure the row is filled out

Reply:
Since its only one column in the row, i will use col-md-12